### ISSUE TITLE
Can't install trustymail - syntax error Non-ASCII character '\xc2'

### ISSUE THREAD

**Issue by upieper** [dhs-ncats/trustymail#87]
Hi - I have problems installing trustymail - error message below. 
My system is running CentOS 7 and python 2.7.5 (pip 18). 

```
> pip install trustymail
Collecting trustymail
  Using cached https://files.pythonhosted.org/packages/95/d2/0567a25b9c125ba04730c2a6e01776d377acbd84bb2c8fe9b76fec44ee67/trustymail-0.5.7-py2.py3-none-any.whl
Collecting pyspf==2.0.11 (from trustymail)
  Using cached https://files.pythonhosted.org/packages/02/90/4c33860df1020963a97e3ba79766e435a983000a5d3387e5502653de3f05/pyspf-2.0.11.tar.gz
Collecting py3dns>=3.1.0 (from trustymail)
  Using cached https://files.pythonhosted.org/packages/b9/46/8ea57e6722fec2384d2de7afee81b892693ae3ed99917756d71dd0027739/py3dns-3.2.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-99blp9/py3dns/setup.py", line 7, in <module>
        import DNS
      File "/tmp/pip-install-99blp9/py3dns/DNS/__init__.py", line 27, in <module>
        from .Base import DnsRequest
      File "/tmp/pip-install-99blp9/py3dns/DNS/Base.py", line 9
    SyntaxError: Non-ASCII character '\xc2' in file /tmp/pip-install-99blp9/py3dns/DNS/Base.py on line 10, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-99blp9/py3dns/
```

**Comment by upieper** [comment]
I have found an issue report for the problem with py3dns: https://bugs.launchpad.net/py3dns/+bug/1774134

I will install python 3 and see whether this is getting resolved. It would be good to add a required python version to the trustymail README file.

**Comment by mcdonnnj** [comment]
Later commits have resolved this and Python 3.6+ is the current requirement.
